lib/page/home/home_page.dart

```diff
@@ -57,38 +65,51 @@ class _HomePageState extends ConsumerState<HomePage> {
 
   @override
   Widget build(BuildContext context) {
+    ref.listen<HomeState>(homeViewModelProvider, (
+      HomeState? previousState,
```

Comment:
same, `previousState` has no usage, use `_`

Reply:
Fixed in https://github.com/chornerman/Survey/pull/108